### PR TITLE
Features/simple app  

### PR DESCRIPTION
**Pull request by neaguolt**
Request to integrate this features to main app
## Routing

### CODE REVIEW

**Review by nofloop — CHANGES_REQUESTED** [review]
All code good , but dont like the padding of the main layout

**Comment by neaguolt** [comment]
Do you mean in the home page layout in the main container :
src/app/layout.tsx line 23

File: src/app/layout.tsx
Comment (nofloop) [inline src/app/layout.tsx:1]: Yes in this file

File: src/app/layout.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default function RootLayout({
       <body>
         <Providers>
           <TopNav></TopNav>
-          <main className='container mx-auto p-10'>
+          <main className='container mx-auto p-20'>
```
Comment (nofloop) [inline src/app/layout.tsx:23]: Yes its perfect now

**Review by nofloop — APPROVED** [review]
Ok to pull

**Pull request merged**